mirrord/operator/src/crd/policy.rs

```diff
@@ -232,6 +237,70 @@ pub struct IncomingNetworkPolicy {
     pub http_filter: HttpFilterPolicy,
 }
 
+/// Outgoing network operations policy that partialy mimics the mirrord `network.outgoing` config.
+#[derive(Clone, Default, Debug, Deserialize, Eq, PartialEq, Serialize, JsonSchema)]
+#[serde(rename_all = "camelCase")]
+pub struct OutgoingNetworkPolicy {
+    /// List of allowed outgoing connections.
+    /// If specified, only connections matching these rules will be permitted.
+    /// Evaluated before `block` rules, so a connection that matches both will be blocked.
+    #[serde(default)]
+    pub allow: Vec<OutgoingRule>,
```

Comment:
I think these `Vec`s should probably be `HashSet`s, we don't want duplicate rules, duplicate ports, duplicate anything in these lists.